pandas/tests/extension/test_integer.py

```diff
@@ -168,6 +168,39 @@ def check_opname(self, s, op_name, other, exc=None):
     def _compare_other(self, s, data, op_name, other):
         self.check_opname(s, op_name, other)
 
+    @pytest.mark.parametrize("dtype", ["Int64", "Int32", "Int16", "Int8"])
+    def test_compare_unequal_to_string(self, dtype):
+        # GH 28930
+        s = pd.Series([1, None], dtype=dtype)
+        result = s == "a"
+        expected = pd.Series([False, False])
+
+        self.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("dtype", ["Int64", "Int32", "Int16", "Int8"])
```

Comment:
use the compare_operators_no_eq_ne fixture to test these things

Reply:
Updated to check the other comparisons as well with `all_compare_operators`